### ISSUE TITLE
Multi node test framework

### ISSUE THREAD

**Issue by 0legIv** [aeternity/elixir-node#417]
Framework to simplify multi nodes testing. It must have such functionalities: 

- [ ] Add new node
- [ ] Add all types of txs to every node
- [ ] Run sync test

**Comment by cytadela8** [comment]
Isn't this actually a duplicate of #307? Maybe we should close #307?

**Comment by 0legIv** [comment]
@cytadela8 This one is a framework, and #307 is a test that uses this framework. So it's not a duplicate, but probably we really can close it if I move the test here, to be more understandable.

**Comment by 0legIv** [comment]
Blocked by #536
